.github/workflows/optimize-test.yml

```diff
@@ -1,16 +1,15 @@
 name: Optimize Model Test
 
 on:
```

Comment:
do we need to keep those commented lines?

Reply:
we can remove them.